agent/src/dist/logging/logback.console.xml

```diff
@@ -9,4 +9,7 @@
   <root level="INFO">
     <appender-ref ref="CONSOLE"/>
   </root>
+
+  <logger name="shadow.org.apache.spark" level="warn" />
+  <logger name="shadow.org.eclipse.jetty" level="warn" />
```

Comment:
Is this leftover logging configuration change from debugging?

Reply:
No this was intentional, as spark and jetty are very verbose and spam the whole log which is not what we are interested in. We want to know if there are any problems, but not why which route was taken to serve the request etc.